src/window.rs

```diff
@@ -183,6 +183,23 @@ impl Window {
         title: &str
     ) -> Result<Window> {
         let event_loop = EventsLoop::new();
+
+        #[cfg(any(target_os = "linux", target_os = "freebsd",
```

Comment:
This block should be moved into a method like `platform_window_init` so that there's not conditionally compiled code in the method body.